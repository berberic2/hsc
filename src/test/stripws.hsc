<HTML><HEAD><TITLE>Strip White Spaces</TITLE></HEAD><BODY>

    a  <$stripws type=both>   b c
    a  <$stripws type=prev>   b c
    a  <$stripws type=succ>   b c
    a  <$stripws type=none>   b c

</BODY></HTML>
