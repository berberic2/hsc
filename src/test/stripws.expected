<HTML><HEAD><TITLE>Strip White Spaces</TITLE></HEAD><BODY>

    ab c
    a   b c
    a  b c
    a     b c

</BODY></HTML>
